### PR TITLE
CORE-4355: Restore "CorDapp library dependency" log output

### PR DESCRIPTION
**Pull request by josephzunigadaly**
Restore the library log message and the tests covering the log messages.

### CODE REVIEW

File: cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/DependencyCalculator.kt [hunk 1]
```diff
@@ -158,6 +158,10 @@ open class DependencyCalculator @Inject constructor(objects: ObjectFactory) : De
             disallowChanges()
         }
 
+        for (library in _libraries) {
+            logger.info("CorDapp library dependency: {}", library.name)
+        }
```
Comment (chrisr3) [inline cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/DependencyCalculator.kt:163]: I think we can chain these staements together:
```kotlin
_libraries.apply {
    setFrom(packageFiles - bundledFiles)
    disallowChanges()
}.forEach { library ->
    logger.info("CorDapp library dependency: {}", library.name)
}
```

**Review by chrisr3 — CHANGES_REQUESTED** [review]
Small Kotliny thing...

File: cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/DependencyCalculator.kt (hunk 1 above)
Comment (josephzunigadaly) [reply cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/DependencyCalculator.kt]: Done

**Review by chrisr3 — APPROVED** [review]
LGTM

**Pull request merged**